scrapy/http/request/rpc.py

```diff
@@ -7,9 +7,13 @@
 import xmlrpc.client as xmlrpclib
 from typing import Any, Optional
 
+import defusedxml.xmlrpc as xml_rpc
```

Comment:
I don't think this alias is much better, as I think the only alias that makes sense here is something like `defusedxml_xmlrpc` and I don't think that's better than just importing `defusedxml.xmlrpc`, unless I'm missing some reasons why it's not possible.

Reply:
Agreed indeed can cause confusion, just pushed the suggested change!